docs/community/governance-review-week.md

```diff
@@ -0,0 +1,220 @@
+# Governance Review Week
+
+# What is it?
+
+It is a week dedicated to the Governance of KOSS.
+
+Our intention behind organizing the Governance Review Week is to help the core team members measure their level of involvement in KOSS. The week also promotes open discussions between Seniors and the Core team members. We want each one of our members to be frank with their batchmates as well as seniors in sharing their opinions and perspective on every aspect of KOSS.
+
+We also review how KOSS performed as a team in the past year. We identify our problems and discuss solutions. We want to make sure we eliminite all forms of toxicity from KOSS (It can be habits of how we do things, or some Core Team members themselves.)
```

Comment:
A Grammarly check might be good on the entire document: `eliminite` -> `eliminate`